flight/test_files/state_machine_test.py

```diff
@@ -15,7 +15,7 @@ async def run_test(_sim: bool) -> None:  # Temporary fix for unused variable
     _sim : bool
```

Comment:
Don't forget to change the variable name in the docstring here and in the function definition